readthedocs/core/resolver.py

```diff
@@ -100,7 +100,7 @@ def base_resolve_path(
         # TODO: remove this when all projects have migrated to path prefixes.
         # Allow users to override their own URLConf
         # This logic could be cleaned up with a standard set of variable replacements
-        if urlconf:
+        if urlconf and not custom_prefix:
```

Comment:
I'm guessing that `Feature.USE_UNRESOLVER_WITH_PROXITO` isn't strictly necessary here. But if you add it, it will:

1) Be easier to find again when we are removing legacy code
2) Theoretically if something needs to be quickly restored, we have the feature flag as a safe-guard.

But totally up to you :+1:

Reply:
oh, this is totally intentional to archive the first point

> With this we would be able to start generating
links using the path-prefix while serving docs with the old proxito implementation.

I can add a comment here.